tests/lib/configs.test.ts

```diff
@@ -11,6 +11,17 @@ describe('Validation enabled and disabled configurations', () => {
 	const setPredicate = s.set(s.number);
 	const tuplePredicate = s.tuple([s.string, s.number]);
 
+	const predicateAndValues = [
```

Comment:
this looks like a minor side change, but I would suggest to do this in a separate PR to "reduce scope", but that's just me.

Reply:
I'm using this array in my new test so I don't think it's out of scope.